charts/gateway-helm/values.tmpl.yaml

```diff
@@ -49,10 +49,10 @@ config:
 
 envoyGatewayMetricsService:
   ports:
-    - name: https
-      port: 8443
+    - name: http
```

Comment:
it looks like this was pointing to the controller runtime metrics endpoint earlier
https://book.kubebuilder.io/reference/metrics
https://github.com/envoyproxy/gateway/blob/d65ab597196bba840c9ec87dca2676d6b0502a77/charts/gateway-helm/templates/envoy-gateway-deployment.yaml#L74

are we disabling that now ?

Reply:
We have collected them together, we have the controller runtime metrics and also our own metrics. They are exposing at 19001 together.

Reply:
do we then need to delete the above listen address in L75 ?

Reply:
let me check.

Reply:
Do not need it anymore.